<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1209</width>
    <height>945</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelPixmapFashion">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>340</y>
     <width>1200</width>
     <height>300</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string>Empty</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelVideoFace">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>620</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelVideoClothes">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>610</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelVideoBody">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>610</y>
     <width>57</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>751</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Missing Person DB</string>
   </property>
   <widget class="QLabel" name="labelBirthDay">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>AGE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PersonADD">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelGender">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gender</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelGender_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Height</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetDB">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>501</width>
      <height>241</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Gender</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Height</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TopColor</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>AGE</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="labelClothesTop">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TopColor</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelGender_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>cm</string>
    </property>
   </widget>
   <widget class="QComboBox" name="TopColor">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Black</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>White</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yellow</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Red</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Green</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>PurPle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Blue</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="Name">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Height">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="FEMALE">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>F</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="MAN">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Birth">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pictureUpload">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>사진 선택</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelClothesTop_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Picture</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelResult">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>840</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>30</y>
     <width>431</width>
     <height>301</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
  </widget>
  <widget class="QLabel" name="resultColor">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>670</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
